src/patternfly/components/Table/table.scss

```diff
@@ -466,8 +466,8 @@
 
   // - Table truncate - Table wrap - Table nowrap - Table fit content - Table wrap - Table break word
   :where([class*='#{$table}']),
-  :where(&) > :is(thead, tbody) {
-    @at-root .#{$table} > thead,
+  :where(&) > :is(thead, tbody),
+  &.#{$table}__thead {
```

Comment:
This compiles to

```
.pf-v6-c-table :where([class*=pf-v6-c-table]),
:where(.pf-v6-c-table) > :is(thead, tbody),
.pf-v6-c-table.pf-v6-c-table__thead
```

Just want to verify that's correct - particularly the last line? Curious when we have `.pf-v6-c-table.pf-v6-c-table__thead` if so.